### PR TITLE
Update github URL

### PR DESCRIPTION
**Pull request by trungleduc**

### CODE REVIEW

**Review by martinRenou — APPROVED** [review]
Thanks!

**Comment by martinRenou** [comment]
Ah wait, the CI failure looks legit:

```
ValueError: package.json for '@jupytercad/jupytercad-app' does not define a 'repository.url' matching the cloned repository 'jupytercad/JupyterCAD'.
```

File: python/jupytercad_app/package.json [hunk 1]
```diff
@@ -23,7 +23,7 @@
   "style": "style/index.css",
   "repository": {
     "type": "git",
-    "url": "https://github.com/jupytercad/jupytercad.git"
+    "url": "https://github.com/jupytercad/jupyterCAD.git"
```
Comment (trungleduc) [inline python/jupytercad_app/package.json:26]: > ValueError: package.json for '@jupytercad/jupytercad-app' does not define a 'repository.url' matching the cloned repository 'jupytercad/JupyterCAD'.

This error looks weird

Comment (martinRenou) [reply python/jupytercad_app/package.json:26]: It seems you have a missing capital letter `jupyterCAD` -> `JupyterCAD`

Comment (trungleduc) [reply python/jupytercad_app/package.json:26]: ah thanks! case-sensitive repo name never is a good idea

**Pull request merged**